README.md

```diff
@@ -1 +1,48 @@
-# 24HrChallenge
+# 24HrChallenge - Anonymous thought - Humza & Sammie
+
+## Description
+
+For this project we were tasked with creating a  place where people can write down their thoughts, whilst remaining anonymous. The following features were required:
+
+* Your app should have a browser client allowing users to write a post with a title, a pseudonym and a body
+* No login should be required to create a post or visit a post
+* When a user hits 'publish', the post should be stored in a database and the client redirected to a show path
+* The user should be able to access their post using that show path even after a server restart
+* Edit and delete functionality is not required
+
+This readme will detail how to set-up and install this application, the technologies used as well as the wins and challenges we came across druring this project. 
+
+## Installation & Usage
+
+### Installation
+
+* Clone or download the repo.
+* cd into server and run the following:
+    * run `docker compose up`
+* Then cd into the client side and run either a `http-server` or a `pyhton server`
```

Comment:
For client, having a package.json which has lite-server would really help during development as it restarts the browser each time you save a change.